saleor/graphql/account/mutations/base.py

```diff
@@ -137,19 +177,26 @@ def perform_mutation(cls, _root, info, **data):
         new_password = data["new_password"]
 
         if not user.check_password(old_password):
-            raise ValidationError({"old_password": "Old password isn't valid."})
+            raise ValidationError(
+                {
+                    "old_password": ValidationError(
+                        "Old password isn't valid.",
+                        code=AccountErrorCode.INVALID_PASSWORD,
+                    )
+                }
+            )
         try:
             password_validation.validate_password(new_password, user)
         except ValidationError as error:
-            raise ValidationError({"new_password": error.messages})
+            raise ValidationError({"new_password": error})
```

Comment:
Shouldn't return error code here?

Reply:
`error` here is a `ValidationError` instance and it should already have error code.